aws/helper.py

```diff
@@ -0,0 +1,112 @@
+import os
+import signal
+import time
+from random import choice
+from string import ascii_lowercase
+
+from .logger import get_logger
```

Comment:
Once the file is moved, you can use `from jinacld_tools.logger import get_logger`